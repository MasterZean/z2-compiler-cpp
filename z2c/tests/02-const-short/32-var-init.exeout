var a: Short = Short{0}
var b: Short = Short{-128}
var c: Short = Short{127}
var d: Short = Short{-129}
var e: Short = Short{128}
var aa: Short = Short{-32768}
var bb: Short = Short{32767}
var cc: Short = Short{-32769}
var ff: Short = Short{32768}
var f: Short = Short{0}
var g: Short = Short{0}
var h: Short = Short{0}
var i: Short = Short{128}
var ii: Short = Short{32768}
var j: Short = Short{97}
var k: Short = Short{127}
var l: Short = Short{128}
var kk: Short = Short{32767}
var ll: Short = Short{32768}
var m: Short = Short{0}
var n: Short = Short{1}
