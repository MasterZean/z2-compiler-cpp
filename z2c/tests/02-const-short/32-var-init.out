int16 a = (int16)(0);
int16 b = (int16)(-128);
int16 c = (int16)(127);
int16 d = -129;
int16 e = 128;
int16 aa = -32768;
int16 bb = 32767;
int16 cc = (int16)(-32769);
int16 ff = (int16)(32768);
int16 f = (int16)(0.0f);
int16 g = (int16)(0.0);
int16 h = (int16)(0u);
int16 i = 128u;
int16 ii = (int16)(32768u);
int16 j = (int16)('a');
int16 k = (int16)(127);
int16 l = (int16)(128);
int16 kk = (int16)(32767);
int16 ll = (int16)(32768);
int16 m = (int16)(false);
int16 n = (int16)(true);
